docs/configuration/overview.rst

```diff
@@ -293,6 +293,10 @@ This section might be relevant to users who wish to *fan-in* (N machines replica
   * ``sink`` constrains each client to a disjoint sub-tree of the sink-side dataset hierarchy ``${root_fs}/${client_identity}``.
     Therefore, the different clients cannot interfere.
 
+* Fan-out replication: 1 ``pull`` job on each target server, each with a corresponding unique ``source`` job on the source server
+
+  * With fan-out replication, the datasets referenced by each ``source`` job are likely not disjoint. This can work if snapshotting and sender-side pruning is done by a separate ``snap`` job. Snapshotting should be disabled in each ``source`` job on the source server and sender-side pruning disabled in the ``pull`` job on each target server.
```

Comment:
Please use separate lines per sentence. Align with spaces to the first character after the `*`, then it works.

Additional remark on content: the reader should keep in mind that if they restore the `source` from one of the `pull` sites, the restored system will not have the replication cursor bookmarks created for the other `pull` jobs targets. Hence, it is not guaranteed that incremental replication to the other `pull` jobs will work after a restore.

Reply:
Thanks! I've added the additional remark as suggested.